src/main/resources/assets/automodpack/lang/ru_ru.json

```diff
@@ -1,30 +1,48 @@
 {
-
-
-  "EVERYTHING HERE NEEDS TO BE RE-TRANSLATED": "TAKE A LOOK AT en_us.json FILE",
-  "THANK YOU": "SO MUCH!",
-
-
-  "gui.automodpack.screen.restart.title.modpack": "Модпак успешно обновлен!",
-
-  "gui.automodpack.screen.restart.button.cancel": "Нет (Вернутся в игру)",
-
-  "gui.automodpack.screen.confirm.title": "❌ Удалить модпак ❌",
-  "gui.automodpack.screen.confirm.description": "Вы удалите все данные модпака.",
-  "gui.automodpack.screen.confirm.secDescription": "Вам нужно будет перезапустить игру.",
-  "gui.automodpack.screen.confirm.thiDescription": "Вы уверены что хотите удалить модпак?",
-  "gui.automodpack.screen.confirm.button.sure": "Да (Удалить и выйти)",
-  "gui.automodpack.screen.confirm.button.cancel": "Нет (Вернутся в игру)",
-
-  "gui.automodpack.screen.loading.title": "Скачиваю обновления...",
-  "gui.automodpack.screen.loading.wait": "Подождите...",
-  "gui.automodpack.screen.loading.download": "Скачиваю...",
-  "gui.automodpack.screen.loading.extract": "Роспичатываю модпак...",
-  "gui.automodpack.screen.loading.finish": "Почти закончил...",
-
-  "gui.automodpack.screen.danger.title": "Внимание! Модпак найден!",
-  "gui.automodpack.screen.danger.description": "Вы хотите скачать обновление для модпака?",
-  "gui.automodpack.screen.danger.secDescription": "Может быть рисковано! Не скачивайте если не доверяете админам сервера!",
-  "gui.automodpack.screen.danger.button.accept": "Скачать",
-  "gui.automodpack.screen.danger.button.cancel": "Не скачивать"
-}
+	"automodpack.restart.full": "Модпак был скачан!",
+	"automodpack.restart.update": "Обновление Модпака было скачано!",
```

Comment:
```suggestion
	"automodpack.restart.update": "Обновление модпака было скачано!",
```

Reply:
In en_us Modpack is spelled with capital `M`